docs/development/affiliated-packages.rst

```diff
@@ -282,17 +282,15 @@ files manually`_ section since this explains what many of the files do.
    license.
 
 #. Take a moment to look over the ``<packagename>/example_mod.py``,
-   ``<packagename>/tests/test_example.py``, ``scripts/script_example``,
-   and ``<packagename>/example_c.pyx`` files, as well as the
-   ``<packagename>/example_subpkg`` directory. These are examples of a
-   pure-python module, a test script, an example command-line script, a
+   ``<packagename>/tests/test_example.py``, and ``<packagename>/example_c.pyx``
+   files, as well as the ``<packagename>/example_subpkg`` directory.
+   These are examples of a pure-python module, a test script, a
```

Comment:
I don't think that's true, why would they refer to the command line scripts twice, per the prior text: ...a test script, an example command-line script... 
Also, it seems like the order of reference to the example matches the order of the file listing
 ``<packagename>/example_mod.py``  = pure-python module
-   ``<packagename>/tests/test_example.py``  =  a test script
``scripts/script_example`` = an example command-line script
-   and ``<packagename>/example_c.pyx`` =  `Cython`_ module
-   ``<packagename>/example_subpkg`` directory = a sub-package

Hence the use of "respectively" at the end of the sentence.